### ISSUE TITLE
Uninstall hook?

### ISSUE THREAD

**Issue by robbiepaul** [dxw/contact-form-7-rate-limiting#2]
If we release this plugin we should think about adding an uninstall hook that removes the `wp_cf7_rate_limiting` table

**Comment by tomdxw** [comment]
Yes, that makes sense. We'll certainly come back to this issue if/when we make an effort to release it on wordpress.org.
